go.mod

```diff
@@ -33,7 +33,11 @@ require (
 	github.com/miekg/dns v1.1.58
 	github.com/minio/minio-go/v7 v7.0.68
 	github.com/mitchellh/mapstructure v1.5.0
+	github.com/oklog/ulid/v2 v2.1.0
 	github.com/olekukonko/tablewriter v0.0.5
+	github.com/openfga/api/proto v0.0.0-20240307175852-df09bc1e8e82
+	github.com/openfga/language/pkg/go v0.0.0-20240307152633-772cdd8b6a9c
+	github.com/openfga/openfga v1.5.0
```

Comment:
This is pulling in lots of stuff, two surprising items are mysql driver and docker client?

Is it possible for us to consume a more specific package rather than the general openfga package.

Not a blocker ofc.

Reply:
I don't think this is possible as the go.mod is in the root of the repo. I could look into which specific dependencies are required by the packages we are using and potentially we can write exclude directives to get rid of mysql and docker etc.

Reply:
i think go mod tidy would have done this for us already if it wasnt needed - i was wondering if we did need to import the main package or whether we can be more specific somehow?